pages/linux/dmidecode.md

```diff
@@ -0,0 +1,23 @@
+# dmidecode
+
+> Display the DMI (some say SMBIOS) table contents in a human-readable format as root user.
```

Comment:
Not sure that `as root user` is needed since you've prefixed all the commands with `sudo` below.

Reply:
Yeah, Actually I'm not sure if I should append `as root user` to the description, as I've added `sudo` to example commands.
Maybe I should say `Display the DMI (some say SMBIOS) table contents in a human-readable format with root privilege.`

I'm confused. a little.